server/apis/user_communication.py

```diff
@@ -6,12 +6,39 @@
 from apis.api_utils import do_and_return_response
 from busniess_layer.logic_controller import LogicController
 from config import config
-from datetime import datetime
+import datetime
 from flask import jsonify
-
+import jwt
+from functools import wraps
 
 app = Flask(__name__)
 socketio = SocketIO(app)
+app.config['SECRET_KEY'] = 'Th1s1ss3cr3t_NOT_FOR_PROD'
+jwt_algorithm = 'HS256'
+
+
+def token_required(f):
```

Comment:
will the token be a constant for now? 
right now I don't have the ability to access the state of the login bloc from the internal blocs of the app, which pose
a problem how to interact with the server